docs/source/gateway/index.rst

```diff
@@ -1,6 +1,1049 @@
+
+    print(response.json())
+
+The response printed to ``stdout`` will be:
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "message": {"role": "assistant", "content": "\n\nYes, this is a test."},
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 7,
+            "total_tokens": 20,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+
+Step 7: Send Requests Using the Fluent API
+------------------------------------------
+
+Here's an example of how to send a chat request using the ``fluent`` API:
+
+.. code-block:: python
+
+    from mlflow.gateway import query, set_gateway_uri
+
+    set_gateway_uri(gateway_uri="http://localhost:5000")
+
+    response = query(
+        "chat",
+        {"messages": [{"role": "user", "content": "What is the best day of the week?"}]},
+    )
+
+    print(response)
+
+**Note:** Remember to change the uri definition to the actual uri of your Gateway server.
+
+The returned response will be in this data structure (the actual content and token values will likely be different):
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "message": {
+                    "role": "assistant",
+                    "content": "\n\nIt's hard to say what the best day of the week is.",
+                },
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 15,
+            "total_tokens": 28,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
+
+
+Step 8: Send Requests Using the Client API
+------------------------------------------
+Here's an example of how to send a completions request using the ``MlflowGatewayClient`` API:
+
+.. code-block:: python
+
+    from mlflow.gateway.client import MlflowGatewayClient
+
+    client = MlflowGatewayClient(gateway_uri="http://localhost:5000")
+
+    response = client.query(
+        "completions_route", {"prompt": "Why don't we ever tell secrets on a farm?"}
+    )
+
+    print(response)
+
+**Note:** Remember to change the uri definition to the actual uri of your Gateway server.
+
+The response to this will have the following structure (and not necessarily the same continuation text):
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "text": "\n\nBecause the potatoes have eyes, the corn has ears, and the beans stalk.",
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 10,
+            "output_tokens": 13,
+            "total_tokens": 23,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/completions",
+        },
+    }
```

Comment:
I think we can remove this and instead tell users that they can also make `MlflowGatewayClient` requests and link them to the MLflow Python Client section